web_src/less/features/heatmap.less

```diff
@@ -1,31 +1,27 @@
 #user-heatmap {
   width: 100%;
-  text-align: center;
+  font-size: 9px;
```

Comment:
If possible, avoid `px` values. They don't scale with zoom, so prove to be an accessibility issue.

Could require an issue against upstream, I haven't checked.

Reply:
Although I do not like the `px` either, but it is what the old component's style.

It's just a quick fix to get old style.

Reply:
Fair enough. Just figured to highlight some drawbacks with this approach, so if someone does the forensics, this comment can be found.